src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -678,6 +711,17 @@ class SceneryWindow final : public Window
             const auto lastTabIndex = _tabEntries.size() - 1;
             const auto lastTabWidget = &widgets[WIDX_SCENERY_TAB_1 + lastTabIndex];
             windowWidth = std::max<int32_t>(windowWidth, lastTabWidget->right + 3);
+
+            for (size_t index = 0; index < _tabEntries.size(); index++)
+            {
+                if (!_tabEntries[index].IsSearch())
+                    continue;
+
+                auto searchTabWidget = &widgets[WIDX_SCENERY_TAB_1 + index];
+                searchTabWidget->left = windowWidth - TabWidth - 2;
+                searchTabWidget->right = windowWidth - 3;
+                break;
+            }
```

Comment:
@fidwell I'm wondering how you expect this bit of code to work with #19373. If it makes sense I can remove these lines and we can just treat this new tab like the other tabs.

Reply:
I think it'd still work. We could pin the new tab to the right still, just on the last row, like this:
![image](https://user-images.githubusercontent.com/5436387/218590414-9d89edab-fc8f-4829-bcee-71816f5aab42.png)

If my PR gets merged before yours I'm happy to help implement this.

Reply:
I think that can work. Is that a mockup or did you get it working like that?

Reply:
It's a mockup.

Reply:
I recommend it to be in the top-right of the tablist.

Reply:
Hmm I gotta think about how to best do that.

Reply:
![image](https://user-images.githubusercontent.com/3720978/218907374-d7d6cd01-3914-4fd4-aa57-1e0c592b6099.png)

Reply:
If it makes sense I can play with the offsets a little bit and get some space between it and the previous tab, what do you think @karst ?